inluxclient/client_query.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2021 InfluxData, Inc. All rights reserved.
+// Use of this source code is governed by MIT
+// license that can be found in the LICENSE file.
+
+package influxclient
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"net/url"
+)
+
+// Dialect defines model for Dialect.
+type Dialect struct {
+	Annotations []string `json:"annotations"`
+	Delimiter   string   `json:"delimiter"`
+	Header      bool     `json:"header"`
+}
+
+// QueryParam is key-value pair for parameters inside the Flux query
+type QueryParam struct {
+	Key   string
+	Value string
+}
+
+// Query defines model for Query.
+type Query struct {
```

Comment:
"model" doesn't really mean that to me FWIW, and this struct isn't part of the public API, so how about this?

```suggestion
// queryBody holds the body for an HTTP query request.
type queryBody struct {
```

Reply:
Ok